lib/matplotlib/ticker.py

```diff
@@ -1991,7 +1991,7 @@ def ge(self, x):
 
 class MaxNLocator(Locator):
     """
-    Select no more than N intervals at nice locations.
```

Comment:
I don't think this is true once you consider the number of ticks that are in the limits.